x-pack/plugins/infra/public/components/navigation/routed_tabs.tsx

```diff
@@ -29,13 +31,7 @@ export class RoutedTabs extends React.Component<RoutedTabsProps> {
           key={`${tab.path}${tab.title}`}
           path={tab.path}
           children={({ match }) => (
-            <EuiTab
-              onClick={() => {
-                return;
-              }}
-              isSelected={match ? true : false}
-              disabled={match !== null}
-            >
+            <EuiTab onClick={noop} isSelected={match !== null}>
```

Comment:
I only now noticed there might be an accessibility issue with nesting the `<Link>` inside of the `<EuiTab>`: The tab itself is a button, but doesn't have any effect when activated. Instead, the `<a>` rendered inside of it needs to be activated. In addition, the tab has two focus states (tab focused and link focused), which might confuse screen readers.

So maybe the best way to avoid that would be to do a `history.push` (which we can also get alongside the `match`) directly in the `onClick` of the `EuiTab`?